modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js

```diff
@@ -130,18 +130,42 @@ function AssetTagsSelector({
 
 		openSelectionModal({
 			buttonAddLabel: Liferay.Language.get('done'),
+			getSelectedItemsOnly: false,
 			multiple: true,
 			onSelect: (dialogSelectedItems) => {
 				if (!dialogSelectedItems?.length) {
 					return;
 				}
 
-				const newValues = dialogSelectedItems.map((item) => {
-					return {
-						label: item.value,
-						value: item.value,
-					};
-				});
+				const [newValues, removedValues] = dialogSelectedItems.reduce(
```

Comment:
Hey @jonathanmccann!

I now this naming was there before but, don't you think that the newValues/addedItems and removedValues/removedItems variable names sound a bit confusing? They look like the same concept to me.

Maybe we can just reuse one of them and declare the variables with `let` instead of const. What do you think about that?

Reply:
I think that would help clean this up and make it more readable. Would you like me to make the changes?

Reply:
sure if you can :smile: